### ISSUE TITLE
Calendar card: Allow choosing default view

### ISSUE THREAD

**Issue by bverkron** [home-assistant/frontend#7038]
## The request

Add an option to set the default view for the Calendar card. Either Monthly, Weekly, or Daily. Current default is Monthly.

**Background:**
The default monthly view is the least valuable to me on my dashboard (mostly due to massive space required to properly see events). The weekly view would be much preferred but each time the dashboard re-loads I have to re-select it manually which reduces a lot of the value of having it on my wall panel. The family can't take a quick glance at the board and see what's coming up today or for the week.

## The alternatives

Community add-ons which I'd rather not get into.

**Comment by zsarnett** [comment]
https://github.com/home-assistant/frontend/pull/6982

**Comment by bverkron** [comment]
Thanks glanced for an existing request but missed it.

**Comment by zsarnett** [comment]
We can leave this open until the PR is merged. So that others don't create issues for it :)

**Comment by bverkron** [comment]
Ah, didn’t notice it was a PR you tagged, thought it was an existing issue / feature request. I need to stop skimming past so quickly between other things today :p

**Comment by eithe** [comment]
Very much appreciated @zsarnett  👍 

If you ever find the time (and find this idea useful for yourself) it would be nice to have an agenda view in addition to week view that would show today's events + the next x upcoming. With week view come Friday/Saturday I have to scroll down to see what's actually up, and at that point I don't really care about those passed events anyway. Something along the lines of this now deprecated custom calendard card https://github.com/ljmerza/calendar-card

**Comment by bverkron** [comment]
I would also love to see this. Was going to request it but already have 4+ bug/feature requests in for calendar 😬

If you submit a feature request outlining your suggestion I'll add my support to it. Not that it would mean much though.

**Comment by eithe** [comment]
@bverkron It's actually already requested; https://github.com/home-assistant/frontend/issues/7064

**Comment by bverkron** [comment]
@zsarnett I see this was merged to the dev branch and feature request was closed. Is there a doc update describing how to use it when it's released? Seems like the doc is done separately in the [home-assistant.io repo](https://github.com/home-assistant/home-assistant.io/blob/current/source/_lovelace/calendar.markdown)?

Taking a guess based on the commits in this repo would it be something like...

```
type: calendar
initial_view: dayGridWeek. # <-- set initial view to weekly view
entities:
  - calendar.errands
  - calendar.family
```

**Comment by zsarnett** [comment]
Shit. Yea I do need to update the docs before 0.116

**Comment by bverkron** [comment]
Sweet. Wasn't sure of the workflow for the HA repos. Am I correct in assuming the front-end doc gets done separately in the [home-assistant.io repo](https://github.com/home-assistant/home-assistant.io/blob/current/source/_lovelace/calendar.markdown) repo?

Also, out of curiosity is there any indication in the GitHub repos of which releases the PRs will end up in? Probably just the next dot release unless it's a critical bug fix?

**Comment by zsarnett** [comment]
Yes the docs is completely separate.

So the release is 0.115.

Bug Fixes that fix bugs that come up in 0.115 go into the next 0.115 dot release.

anything else go into the next Major dot release (in our case just Major release) 0.116
